FF116 Intl.PluralRules - NumFormat v3 (#28097)
mdn/content: files/en-us/web/javascript/reference/global_objects/intl/pluralrules/select/index.md

### Return value

A string representing the pluralization category of the `number`.

> This can be one of `zero`, `one`, `two`, `few`, `many` or `other`.

Reviewer: ```suggestion
This can be one of `zero`, `one`, `two`, `few`, `many`, or `other`.
```

## Description

> This function selects a pluralization category according to the locale and formatting options of a {{jsxref("Intl.PluralRules")}} object.
> These options are set in the [`PluralRules` constructor](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules).

Reviewer: ```suggestion
This function selects a pluralization category according to the locale and formatting options of an {{jsxref("Intl.PluralRules")}} object.
These options are set in the [`Intl.PluralRules()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules) constructor.
```

## Examples

### Using select()

> First define the `Intl.PluralRules` object using [constructor](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules) locale and options parameters.

Reviewer: ```suggestion
First, create an `Intl.PluralRules` object, passing the appropriate `locales` and `options` parameters.
```
